profile.html

```diff
@@ -51,17 +51,17 @@ <h1>Quiz App</h1>
             </svg>
             <h2>Jane Doe</h2>
           </div>
-          <div class="profile--info">
+          <div class="profile__info">
```

Comment:
You can also use `section` elements to create sections inside a parent `section`.

Reply:
Cool that is a great new learning, to nest sections!